wca-guidelines.md

```diff
@@ -203,7 +202,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-B><blindfolded><blindfoldedsolving> Article B: Blindfolded Solving
 
-- B1+) [REMINDER] The competitor must use a puzzle without logos (see [Regulation 3l](regulations:regulation:3l)), textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.
+- B1+) [REMINDER] Other than any permitted logos (see [Regulation 3l](regulations:regulation:3l)), the competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.
```

Comment:
I think the first sentence is wordy and somewhat difficult to parse. How about this?

```suggestion
- B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving. Exception: For 3x3x3 Blindfolded and 5x5x5 Blindfolded, the puzzle may have any logos permitted by [Regulation 3l](regulations:regulation:3l).
```

Reply:
Hmm, I'm not sure if it is really an improvement - it seems a bit weird to give an exception that applies for half of the events

Reply:
Thanks for the response! I see where you're coming from, but I wouldn't view it that way. The focus of the guideline is the prohibition of the vast majority of "features that distinguish similar pieces", and logos for 3BLD/5BLD are a rare case of such distinguishable features being allowed. So I'd say calling it an exception is appropriate.

Under these proposed changes, Article B does not mention that logos are allowed for 3BLD/5BLD but not 4BLD. I would guess that most competitors look to Article B when they want to learn about the blindfolded regulations. But unless they happen to jump around the regulations by clicking 3l and leaving Article B right as they've started reading it, they will come out of it fully unaware that only *some* blindfolded events allow logos. Name-dropping 3BLD and 5BLD makes it clear to such readers that only some blindfolded events allow logos.

As far as the wording goes, I think listing the exception at the end of the guideline makes for a clearer reading experience. The reader doesn't know what "Other than any permitted logos" is meant to communicate until they're halfway through the guideline, and then they'll have to go back and reread the exception at the start of the guideline now that they have the proper context for it. That's a moderately confusing read. On the other hand, if the exception is listed at the end of the guideline, it's immediately obvious which rule it is an exception to, and the use of the word "Exception" primes the reader for an important exception to the guideline they just read.